website/upgrade-guides/0-11.html.markdown

```diff
@@ -143,8 +144,9 @@ With the above configuration, any `aws` provider resources in the module
 `module.example-use1` and the us-west-1 provider configuration for
 `module.example-usw1`.
 
-When only default (non-aliased) providers are in use, automatic inheritance
-to child modules is still supported.
+When only default (non-aliased) providers are in use, and not explicitly
+declared in the module, automatic inheritance to child modules is still
```

Comment:
"the module" feels unclear here... suggest instead "...and not explicitly declared in a child module, automatic inheritance into that module is still supported".

Reply:
That sounds to me like it's all or nothing. Maybe reference these as singular? (pushing...)